src/main/java/seedu/exercise/ui/MainWindow.java

```diff
@@ -77,55 +81,20 @@ public MainWindow(Stage primaryStage, Logic logic) {
         setWindowDefaultSize(logic.getGuiSettings());
         primaryStage.setTitle("ExerHealth");
 
-        setAccelerators();
-
         helpWindow = new HelpWindow();
     }
 
     public Stage getPrimaryStage() {
         return primaryStage;
     }
 
```

Comment:
The accelerators present in the MainWindow is removed? Reason for it being removed? Was the reported bug by JavaFX resolved?

Reply:
We are not using the help menu item anymore so i removed it

Reply:
ok